crates/ark/src/lsp/completions/sources/unique/string.rs

```diff
@@ -115,8 +131,13 @@ mod tests {
             let res = completions_from_string(&context).unwrap();
             assert_match!(res, Some(items) => { assert!(items.len() == 0) });
 
-            // Check one level up too
-            let res = completions_from_unique_sources(&context, ParameterHints::Enabled).unwrap();
+            // Check for same result when consulting (potentially all) unique sources
+            let state = WorldState::default();
+            let builder =
+                crate::lsp::completions::builder::CompletionBuilder::new(&context, &state);
+            let unique_sources = UniqueCompletionsSource;
+            let res = unique_sources.provide_completions(&builder).unwrap();
+
```

Comment:
I think this is the only test that required more than the most superficial of tweaks.